### ISSUE TITLE
Weekly to-do list # 1

### ISSUE THREAD

**Issue by ghost** [qTox/qTox#2321]
A few days ago I created an issue with suggestions how to bring qTox forward. Unfortunately, there was no really discussion about it.

Now, in consultation with @tux3, I will start the 'Weekly to-do list'.
Every week a new thread will be created with some open issues that should be done after the week. The goal of it is to work as many issues as possible off. Perhaps the motivation of some developers will increase with a goal setting (manage all items on the list every week). Also, it can be a starting point for new developers as well. _Actually, it is just a try - if it doesn't bring the desired effect we will stop the experiment._

Needless to say, every developer can fix the issues he would like to work on and every help is very welcome. The suggested issues from the weekly to-do list are, as mentioned, only an extra that will (hopefully) be done after the week is over for progress.

If you have some additional ideas how to bring qTox forward, just post it -> https://github.com/tux3/qTox/issues/2299

Let's start the game and see for what the Tox community is capable of! :wink:

---

**_Weekly to-do list # 1 - Should be done until October 4, 2015:**_
- [ ] https://github.com/tux3/qTox/issues/2199
- [ ] https://github.com/tux3/qTox/issues/2288
- [ ] https://github.com/tux3/qTox/issues/2187

_Every developer is very welcome to work on it! Sometimes also non-developer can help a lot with finding the cause for some bugs._

---

**Comment by ghost** [comment]
No effect, thus closing.

**Comment by agilob** [comment]
I guess people don't want to work with others who come out of nowhere and try to organize work for next days. tux3 is out of the loop in this project for a very long time. We are waiting with 3PRs for him to confirm or reject or test changes... and we're waiting. 1st and 2nd issue have **really** low priority, are on the bottom of a list. Third issue, for me is invalid. Author didn't respond to question, personally I would close it.

**Comment by ProMcTagonist** [comment]
Why is tux3 being waited on? After he returned from his last absence authority was reorganized to prevent him from holding back PRs with absence, correct?

**Comment by tux3** [comment]
> We are waiting with 3PRs for him to confirm or reject or test changes...

Really? There's an organization problem then, I didn't know some PRs were waiting on me.
My 1st priority right now was and is still getting the new AV ready, tested and merged.
I pushed a partially working version in a branch, with only group calls needed before it's ready, and I'm going to finish it now.

I found the PRs assigned to me now, so I'll take a look at it. I suppose I must have gotten a notification for it, but it was lost in the sea of new issues and comments, my mistake.
